dist/api_data.py

```diff
@@ -1286,13 +1286,29 @@ def __cmp__(self, other):
         if read timestamp is earlier than oldest timestamp,
         read timestamp will be rounded to oldest timestamp''',
         type='boolean'),
```

Comment:
Since "round_to_oldest" is an existing parameter, I have not removed it, we will deprecate it. Server-side changes have to be done to switch the usage from "round_to_oldest" to "roundup_timestamps=(read=true)". After the server side changes, we can remove this.